credible/cli-reference.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: 'pcl Reference'
+description: 'Reference documentation for all `pcl` commands, options, and parameters'
+---
+
+<Note>
+New to `pcl`? Start with [Using pcl](/credible/cli-overview) to understand how to use it in your workflow.
+</Note>
+
+## Global Options
+
+```bash
+Options:
+  -j, --json      Output in JSON format
+  -h, --help      Print help
+  -V, --version   Print version
+```
+
+## Commands
+
+Commands are listed in the order they are usually used in the assertion development workflow.
+
+### `auth`
+
+Manage authentication with your Credible Layer dApp account.
+
+```bash
+Authenticate `pcl` with your Credible Layer dApp account
+
+Usage: pcl auth [OPTIONS] <COMMAND>
+
+Commands:
+  login   Login to `pcl` using your wallet
+  logout  Logout from `pcl`
+  status  Check current authentication status
+  help    Print this message or the help of the given subcommand(s)
+
+Options:
+  -u, --auth-url <AUTH_URL>  Base URL for authentication service [env: PCL_AUTH_URL=] [default: https://dapp.internal.phylax.systems]
+  -h, --help                 Print help
+```
+
+#### `auth login`
+
+Login to `pcl` using your wallet. Opens a browser window for wallet authentication.
+
+#### `auth logout`
+
+Logout from `pcl` and clear local authentication token.
+
+#### `auth status`
+
+Check current authentication status.
+
+### `test`
+
+Run tests for your assertions.
+
+```bash
+Run tests
+
+Usage: pcl test [OPTIONS] [PATH]
+
+Options:
+  -h, --help  Print help (see more with '--help')
+
+Common options:
+  -v, --verbosity...            Verbosity level of the log messages. Pass multiple times to increase verbosity (e.g. -v, -vv, -vvv)
+  -q, --quiet                   Do not print log messages
+      --match-path <GLOB>       Only run tests in source files matching the specified glob pattern
+      --match-test <REGEX>      Only run test functions matching the specified regex pattern
+      --match-contract <REGEX>  Only run tests in contracts matching the specified regex pattern
+```
+
+<Note>
+`pcl test` supports all Forge test options. See the [Forge test documentation](https://book.getfoundry.sh/reference/forge/forge-test) for additional options.
```

Comment:
it's a nit, but we should say that it uses forge undeanth, to illustrate that it's the same code. Not that it supports all forge, but it **is** forge.

We should also make a note that it's an older version of forge and newer features might not be available. We periodically update the forge version to get. the new features.